### PR TITLE
Another CSS update

### PR DESCRIPTION
**Pull request by janekteo**
Updated Steps and Result pages
Updated contacts page
Fixed secondary button design
Removed duplicate paragraph in Intro page
Added additional ARIA tags

### CODE REVIEW

File: src/components/Emergency.jsx [hunk 1]
```diff
@@ -16,15 +16,21 @@ class Emergency extends Component {
...
-					Emergency Contact Details</Button>
-				<Button onClick={this.next} className="btn btn-primary btn-lg btn-block" role="button">Restart
+			<div className="col-sm-12 col-md-10 offset-md-1">
+				<h1 className="text-center mt-5 mb-4">Call your nearest emergency department.</h1>
+				<br/>
+				<p>
+					They will ask more questions to assess you further. You will be brought to the COVID-19 Isolation
+					Area for admission or the Resuscitation Area of the hospital if you don’t want to be admitted.
+				</p>
+				<p>
```
Comment (nmenego) [inline src/components/Emergency.jsx]: conflict ata? I've updated the left already.

Comment (janekteo) [reply src/components/Emergency.jsx]: oh. wait. lemme check

File: src/components/StepZero.jsx [hunk 2]
```diff
@@ -34,40 +34,35 @@ class StepZero extends Component {
...
+						<li><a href="https://www.doh.gov.ph/2019-nCoV" role="link">Department of Health</a> – Learn about
 							COVID-19
 							and actions being taken to protect the health of Filipinos
 						</li>
 						<li>Department of Health’s official social media channels:
 							<ul>
-								<li><a href="https://www.facebook.com/OfficialDOHgov">Facebook</a></li>
-								<li><a href="https://twitter.com/DOHgovph">Twitter</a></li>
-								<li><a href="http://bit.ly/DOHPhCOVID-19">Viber</a></li>
+								<li><a href="https://www.facebook.com/OfficialDOHgov" role="link">Facebook</a></li>
```
Comment (nmenego) [inline src/components/StepZero.jsx]: what's `role=link` for?

Comment (janekteo) [reply src/components/StepZero.jsx]: ARIA tags. since "Facebook" lang ang content ng <a>

Comment (janekteo) [reply src/components/StepZero.jsx]: tag

Comment (nmenego) [reply src/components/StepZero.jsx]: TIL

**Pull request merged**